docs/podman-generate.1.md

```diff
@@ -11,10 +11,11 @@ The generate command will create structured output (like YAML) based on a contai
 
 ## COMMANDS
 
-| Command  | Man Page                                            | Description                                                                  |
-| -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
-| kube     | [podman-generate-kube(1)](podman-generate-kube.1.md)| Generate Kubernetes YAML based on a pod or container.                        |
-| systemd  | [podman-generate-systemd(1)](podman-generate-systemd.1.md)| Generate a systemd unit file for a container.                        |
+| Command | Man Page                                                   | Description                                                                        |
+|---------|------------------------------------------------------------|------------------------------------------------------------------------------------|
+| kube    | [podman-generate-kube(1)](podman-generate-kube.1.md)       | Generate Kubernetes YAML based on a pod or container.                              |
+| systemd | [podman-generate-systemd(1)](podman-generate-systemd.1.md) | Generate a systemd unit file for a container. Not supported for the remote client. |
+
 
```

Comment:
As long as you're waiting before push: how about:
```diff
- Generate a systemd unit file for a container
+ Generate systemd unit file(s) for a container or pod
```

Reply:
Good catch, thanks!